### DISCUSSION TITLE
New version for mac?

### DISCUSSION THREAD

**Discussion by UsamaBinKhalid18** [subhra74/xdm#892] [Q&A]
Was unable to install 7.2.8(there must be a problem with the installer). when can we expect new version for mac. I am currently using 7.2.7.

**Comment by hirwablessing** [comment]
> Was unable to install 7.2.8(there must be a problem with the installer). when can we expect new version for mac. I am currently using 7.2.7.

@UsamaBinKhalid18 how did you manage to install the app on MacOs Monterey?

**Reply by UsamaBinKhalid18** [reply]
Its been a long time. I don't exactly remember. I tried using different installers but they didn't work. In the end I think I got it through brew:
`brew install --cask xdm`
